graphql/handler/transport/websocket.go

```diff
@@ -126,6 +127,7 @@ func (c *wsConnection) init() bool {
 }
 
 func (c *wsConnection) write(msg *operationMessage) {
+	//fmt.Println("write", msg.ID, msg.Type, string(msg.Payload))
```

Comment:
👀

Reply:
I was going to leave them in, I end up adding them every time I need to debug websockets.

I didnt want to go down the path of adding low overhead configurable logging. what do you think?